app/models/gift_card_detail.rb

```diff
@@ -29,6 +29,7 @@ class GiftCardDetail < ApplicationRecord
 
   # TODO(jmckibben): Being used in sellers_helper N times. Ideally we'd combine
   # into a single query
+  #TODO change to created_at
```

Comment:
Could you add a space before the TODO, and add your name after it like the above?